src/pages/index.tsx

```diff
@@ -21,16 +31,103 @@ import tClientNext from "../tClientNext";
 import PublicUser from '../shared/publicModels/PublicUser';
 import PublicGroup from '../shared/publicModels/PublicGroup';
 import { MdVideocam } from 'react-icons/md';
+import { HSeparator } from 'horizon-ui/components/separator/Separator';
+import { IYuanjianUser } from 'shared/user';
+import { toast } from "react-toastify";
 
 const AppIndex: NextPageWithLayout = () => {
   const user = useUserInfo();
+  if(!user.name){
+    return <Box paddingTop={'80px'}><SetName u = {user} /><Meetings /></Box>
+  };
   return <Box paddingTop={'80px'}><Meetings /></Box>;
 }
 
 AppIndex.getLayout = (page) => <AppLayout>{page}</AppLayout>;
 
 export default AppIndex;
 
+function SetName({u} : {u:IYuanjianUser}) {
+  const textColor = useColorModeValue('navy.700', 'white');
+  const brandStars = useColorModeValue('brand.500', 'brand.400');
+  const [isOpen, setOpen] = useState(true);
+  const [name, setName] = useState('');
+  
+  const handleSubmit = async () => {
+    if (name) {
+      const updatedUser: IYuanjianUser = {
+        id: u.id,
+        pinyin: name,
+        name: name,
+        email: u.email,
+        roles: u.roles,
+        clientId: u.clientId,
+      };
+
+      tClientBrowser.user.updateProfile.mutate(updatedUser).then(
+        res => {
+          if (res === "ok") {
+            console.log("user update succeeded");
+            setOpen(false);     
+          }
+        }
+      ).catch(e => toast.error(e.message, { autoClose: false }))
+    };
+  };
+
+  return (
+    <Modal isOpen={isOpen} onClose={() => setOpen(false)}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>新用户登录</ModalHeader>
+        <ModalBody>
+          <Text>
+            请填写个人中文全名
+          </Text>
+          <Box mt={4}>
+            <Flex align='center' mb='25px'>
+              <HSeparator />
+            </Flex>
+            <FormControl>
+              <FormLabel display='flex' ms='4px' fontSize='md' fontWeight='500' color={textColor} mb='8px'>
```

Comment:
let's remove styling tags as many as possible, to rely on default styling as much as possible.

Reply:
the same for all other components below and elsewhere.